src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv400.cql

```diff
@@ -88,6 +93,12 @@ valueset "Substance use disorder": '2.16.840.1.113883.3.464.1003.106.12.1004'
 // [See value set in VSAC](https://vsac.nlm.nih.gov/valueset/2.16.840.1.113883.3.464.1003.106.11.1010/expansion)
 valueset "Substance Abuse": '2.16.840.1.113883.3.464.1003.106.11.1010'
 
+// [See value set in VSAC](https://vsac.nlm.nih.gov/valueset/2.16.840.1.113762.1.4.1146.1008/expansion) 
+valueset "Opioid overdose and poisoning (ICD10CM)": '2.16.840.1.113762.1.4.1146.1008'
```

Comment:
Just curious-- is there a public reference for these OIDs? I tried looking them up in the NLM VSAC but I don't have an account :(